db/crud.go

```diff
@@ -337,6 +345,10 @@ func (db *Database) GetDelta(docID, fromRevID, toRevID string) (delta *RevisionD
 			return nil, &redactedBody, nil
 		}
 
+		if toRevision.IsRemoval() {
+			return nil, nil, base.HTTPErrorf(404, "missing")
+		}
+
```

Comment:
We should add unit tests for the delta cases when either fromRevision or toRevision have IsRemoval()==true.

Reply:
Unit tests for the delta cases when either fromRevision or toRevision have IsRemoval()==true are included